404.md

```diff
@@ -6,6 +6,15 @@ heading_anchors: false
 nav_exclude: true
 ---
 
-# OpenSearch cannot find that page. 
+## We can't seem to find the page you're looking for. 
+
+Perhaps we moved something around. Here are some of our most popular pages: 
```

Comment:
Suggested text: Maybe our homepage or one of the popular pages listed below can help.